site/app/templates/forum/CreatePost.twig

```diff
@@ -32,7 +32,7 @@
 
     {% endif %}
 
-    <pre class='pre_forum'><span class="post_content pre-wrap" >{% include "forum/RenderPost.twig" with { "post_content": post_content, "render_markdown": render_markdown } only %}</span></pre>
+    <pre class='pre_forum'><span class="post_content pre-wrap {% if render_markdown == 1 %}forum_markdown{% endif %}" >{% include "forum/RenderPost.twig" with { "post_content": post_content, "render_markdown": render_markdown } only %}</span></pre>
```

Comment:
Can we make all of the list styling based off the `.post_content` class? Not sure we would ever be in the case where we would want different styling of lists in posts based on whether markdown is used or not, though it is not possible to create lists right now otherwise.